Doc/library/ast.rst

```diff
@@ -319,7 +319,7 @@ and classes for traversing abstract syntax trees:
       node = YourTransformer().visit(node)
 
 
-.. function:: dump(node, annotate_fields=True, include_attributes=False)
+.. function:: dump(node, annotate_fields=True, include_attributes=False, *, indent=None)
```

Comment:
Is the `, *,` part here intentional? It doesn't match the definition for the actual function.

Reply:
Thank you. It should be added also in the code.